web/src/settings_panel_menu.js

```diff
@@ -66,11 +67,34 @@ export class SettingsPanelMenu {
     show() {
         this.$main_elem.show();
         const section = this.current_tab;
+        const user_settings_tab = this.current_user_settings_tab;
         if (two_column_mode()) {
             // In one column mode want to show the settings list, not the first settings section.
-            this.activate_section_or_default(section);
+            this.activate_section_or_default(section, user_settings_tab);
         }
         this.$curr_li.trigger("focus");
+        this.org_user_settings_toggler = components.toggle({
+            html_class: "fuck",
+            child_wants_focus: true,
+            values: [
+                {label: $t({defaultMessage: "Users"}), key: "active-users-admin"},
+                {
+                    label: $t({defaultMessage: "Deactivated users"}),
+                    key: "deactivated-users-admin",
+                },
+                {label: $t({defaultMessage: "Invitations"}), key: "invites-list-admin"},
+            ],
+            callback: (_name, key) => {
+                browser_history.update(`#organization/user-list-admin/${key}`);
+                this.set_user_settings_tab(key);
+                $(".user-settings-section").hide();
+                $(`[data-user-settings-section="${CSS.escape(key)}"]`).show();
+            },
+        });
+        $("#admin-user-list .tab-container").append($(this.org_user_settings_toggler.get()));
+        if (section === "user-list-admin") {
+            this.org_user_settings_toggler.goto(user_settings_tab);
```

Comment:
Is this needed? I guess call to `goto` in `activate_section_or_default` should be enough.

Reply:
It was needed when activate_section_or_default was called before, changed the order of things.